app/controllers/returns/ReturnsCheckYourAnswersController.scala

```diff
@@ -91,21 +90,6 @@ class ReturnsCheckYourAnswersController @Inject()(
           .map(_ => Redirect(routes.ReturnsCheckYourAnswersController.onPageLoad()))
     }
 
-  /** TODO quick fix for user removing any previous credit claim
-    * @return true if the user is claiming a credit
-    */
-  private def ensureCreditAnswersConsistency(userAnswers: UserAnswers, pptReference: String)(implicit hc: HeaderCarrier)
```

Comment:
Could removing this break returns submission where user has url hopped? (Can't quite remember why it was added)